app/Notifications/FailedS3Event.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Notifications\Notification;
+use \Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Support\Facades\Log;
+
+class FailedS3Event extends Notification
+{
+    private $event;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct($event)
+    {
+        $this->event = $event;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via()
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail()
+    {
+        Log::info('Sending email to: ' . config('mail.event_notification') . ', path: ' . $this->event['path']);
+        return (new MailMessage)
+            ->subject('Failed to backup content/asset')
+            ->markdown('templates.emails.failed-s3-event', [
+                'event_date' => date('F jS, Y h:i:s A'),
```

Comment:
I think this should just be standard iso format -- in python there's a shortcut for that...

Reply:
it is to be displayed in email, so should be readable right
<img width="683" alt="image" src="https://user-images.githubusercontent.com/17048517/194600769-f572bf21-c762-4fd8-a095-114275ca8f7f.png">

Reply:
Yeah it's fine -- I was just suggesting using the ISO standard like `2021-07-27T16:02:08.070557` (https://www.geeksforgeeks.org/isoformat-method-of-datetime-class-in-python/) because this email is going to come to me (sysadmin@datamermaid.org -- or see above comment, maybe should be >1 address?) and we will want to then be able to search logs and so forth. So human-readable yes but more specifically dev-readable. :-)